packages/block-library/src/query/index.php

```diff
@@ -48,7 +48,7 @@ function render_block_core_query( $attributes, $content, $block ) {
 			$content           = substr_replace(
 				$content,
 				'<div
-					class="wp-block-query__enhanced-pagination-navigation-announce"
+					class="wp-block-query__enhanced-pagination-navigation-announce screen-reader-text"
```

Comment:
Let's also remove this class, which is unnecessary now.

```suggestion
					class="screen-reader-text"
```

If someone needs it, we'll add it back.